composer.json

```diff
@@ -50,7 +50,8 @@
 		"classmap": [ "classes/" ]
 	},
 	"suggest": {
-		"ext-mbstring": "Provides support for UTF-8 strings"
+		"ext-mbstring": "Provides support for UTF-8 strings",
+		"atoum/stubs": "Provides stubs for atoum that will help ide for autocompletion"
```

Comment:
```diff
- ide
+ IDE
```
Actually, the whole sentence is not very clearn.
What about:
> Provides IDE support (like autocompletion)

This is simple but it's understandable.